dev-cpp/gtest/metadata.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>plevine457@gmail.com</email>
+		<name>Peter Levine</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
 	<upstream>
 		<doc lang="en">https://code.google.com/p/googletest/w/list</doc>
```

Comment:
the URL is wrong

Reply:
@jonasstein OK.  In fairness, it does correctly redirect to https://github.com/google/googletest